docs/developer-docs/latest/developer-resources/database-apis-reference/rest/relations.md

```diff
@@ -0,0 +1,238 @@
+```
+
+sending the following in the request body of a PUT or POST request will create a third relation of `id` `3` and position it before the relation with `id` `2`:
+
+```json
+categories: {
+  connect: [
+    { id: 3, position: { before: 2 } }, // It should be placed before relation id=2
+  ]
+}
+```
+
+:::
+
+::: tab Combined example
+
+Sending the following example in the request body of a PUT or POST request updates multiple relations:
+
+```json
+categories: {
+  connect: [
+    // By default positional info will be end: true
+    { id: 6, position: { after: 1} },    // It should be after relation with id 1
+    { id: 7, position: { before: 2 }},   // It should be before relation with id 2
+    { id: 8, position: { end: true }},   // It should be at the end
+    { id: 9 },                           // It should also be at the end
+    { id: 10, position: { start: true }} // It should be at the start
+  ]
+}
+```
+
+The resulting database record could be the following:
+
+```json
+categories: [
+  { id: 10, order: 1},
+  { id: 1, order: 2},
+  { id: 6, order: 3},
+  { id: 7, order: 4},
+  { id: 2, order: 5},
+  { id: 8, order: 6},
+  { id: 9, order: 7}
+]
+```
+
+:::
+
+::::
+
+## `disconnect`
+
+Using `disconnect` in the body of a request performs a partial update, removing specified relations.
+
+`disconnect` accepts either a shorthand or a longhand syntax.
+
+### Shorthand syntax
+
+With the shorthand syntax, pass an array of ids of relations to be removed.
+
+**Example:**
+
+Passing the following example of a body request removes the categories with `id` `4` from the content type's relations:
+
+```json
+categories: {
+  disconnect: [4]
+}
+```
+
+### Longhand syntax
+
+With the longhand syntax, pass an array of objects with the `id` property for each relation to remove.
+
+**Example:**
+
+Passing the following example of a body request removes the categories with the id 4 from the content type's relations:
+
+```json
+categories: {
+  disconnect: [
+    { id: 4 }
+  ]
+}
+```
+
+## `set`
+
+Using `set` performs a full update, replacing all existing relations with the ones specified, in the order specified.
+
+`set` accepts a shorthand or a longhand syntax.
+
+### Shorthand syntax
+
+With the shorthand syntax, pass an array of ids of relations to be set.
+
+**Example:**
+
+Passing the following example of a body request will override all existing relations previously defined for `categories` and replace them only with `id` `2` and `4`:
+
+```json
+{
+  data: {
+    categories: { set: [2, 4] }
+  }
+}
+```
+
+### Longhand syntax
+
+With the longhand syntax, pass an array of objects with the `id` property for each relation to set.
+
+**Example:**
+
+Passing the following example of a body request will override all existing relations previously defined for `categories` and replace them only with `id` `2` and `4`:
+
+```json
+{
+  data: {
+    categories: { 
+      set: [
+        { id: 2 },
+        { id: 4 }
+      ]
+    }
+  }
+}
+```
```

Comment:
Would it be interesting to say somewhere that `connect` and `disconnect` can be sent together and that `set` is supposed to be used alone?